automation/taskcluster/lib/tasks.py

```diff
@@ -470,21 +470,77 @@ def craft_upload_apk_nimbledroid_task(self, assemble_task_id):
             dependencies=[assemble_task_id],
         )
 
-    def craft_raptor_task(self, signing_task_id, mozharness_task_id, variant):
+    def craft_raptor_speedometer_task(self, signing_task_id, mozharness_task_id, variant, gecko_revision):
+        return self._craft_raptor_task(
+            signing_task_id,
+            mozharness_task_id,
+            variant,
+            gecko_revision,
+            name_prefix='raptor speedometer',
+            description='Raptor Speedometer on the Reference Browser',
+            test_name='raptor-speedometer',
+            job_symbol='sp',
+        )
+
+    def craft_raptor_speedometer_power_task(self, signing_task_id, mozharness_task_id, variant, gecko_revision):
+        return self._craft_raptor_task(
+            signing_task_id,
+            mozharness_task_id,
+            variant,
+            gecko_revision,
+            name_prefix='raptor speedometer power',
+            description='Raptor Speedometer power on the Reference Browser',
+            test_name='raptor-speedometer',
+            job_symbol='sp',
+            group_symbol='Rap-P',
+            extra_test_args=[
+                "--power-test",
+                "--page-cycles 5",
+                "--host HOST_IP",
+            ],
+        )
+
+    def craft_raptor_tp6m_task(self, signing_task_id, mozharness_task_id, variant, gecko_revision):
+        # XXX Temporary situation while `./mach raptor-test` exposes the remaining chunks
+        chunk = 1
```

Comment:
I think the chunks are not chunked like we have for unittests- these are hardcoded, but tp6m-1 -> tp6m-10 will all have definitions in raptor.zip

Reply:
Okay. Should I remove the chunk logic, the?

Reply:
I would, or rename it.  If we keep it, put a comment in something like "we are not chunking dynamically, but we have hardcoded jobs that are numbered and using chunk is a way to loop over and schedule these jobs easier"

Reply:
Okay. I went with the latter!